core/src/main/java/com/sequenceiq/cloudbreak/service/blueprint/BlueprintService.java

```diff
@@ -162,7 +161,7 @@ public Blueprint create(Workspace workspace, Blueprint blueprint, Collection<Map
     }
 
     public PlatformRecommendation getRecommendation(Long workspaceId, String blueprintName, String credentialName,
-        String region, String platformVariant, String availabilityZone, CdpResourceType cdpResourceType) {
+            String region, String platformVariant, String availabilityZone, CdpResourceType cdpResourceType) {
```

Comment:
a small thing, but there is a plus tab character

Reply:
Hm that is strange, my IDEA says that this  the right formatting and it has changed it automatically